### PR TITLE
Update benchmarks

### PR DESCRIPTION
**Pull request by JamesKyburz**
Here are the new benchmarks.

### CODE REVIEW

**Comment by aichholzer** [comment]
@JamesKyburz I see new benchmark data, but a lot of the module versions are wrong, as in out of date. Would you mind updating all modules to their latest version and re-running the tests?

**Comment by mcollina** [comment]
@JamesKyburz can you please include #79 and rebase? I think we should land this and update results.

**Comment by JamesKyburz** [comment]
> @JamesKyburz can you please include #79 and rebase? I think we should land this and update results.

Yes, I will can rebase and update this PR tomorrow.

**Comment by JamesKyburz** [comment]
Updated!

**Comment by aichholzer** [comment]
@JamesKyburz -How are you generating the new table?
A lot of the module versions are old and should be updated as well, since we are also updating the numbers they can do.

**Comment by JamesKyburz** [comment]
@aichholzer updated PR again.

You were right, I had run `npm update` so not everything was updated to latest....

Now I ran `npx npm-check-updates -u`

After 2 runs the table is generated with `./benchmark.js compare -t --commandlineMdTable | pbcopy`

File: README.md [hunk 1]
```diff
@@ -44,33 +44,33 @@ benchmark [arguments (optional)]
...
+| egg.js                   |         | ✓      | 16048.0    | 6.13    | 5.33          |
+| express-route-prefix     | 4.16.3  | ✓      | 5764.6     | 17.03   | 2.01          |
+| express-with-middlewares | 4.16.3  | ✓      | 20158.4    | 4.85    | 7.33          |
+| express                  | 4.16.3  | ✓      | 26481.6    | 3.69    | 4.15          |
+| fastify-big-json         | 1.6.0   | ✓      | 8361.2     | 11.69   | 96.40         |
+| fastify                  | 1.6.0   | ✓      | 45846.4    | 2.11    | 7.17          |
+| foxify                   | 0.10.6  | ✓      | 46608.0    | 2.08    | 6.63          |
+| hapi                     | 17.5.1  | ✓      | 26497.6    | 3.70    | 4.16          |
+| koa-router               | 7.4.0   | ✓      | 33054.4    | 2.95    | 5.13          |
+| koa                      | 2.5.1   | ✗      | 35796.8    | 2.72    | 5.63          |
```
Comment (aichholzer) [inline README.md]: Current version is `2.6.2`

File: README.md [hunk 2]
```diff
@@ -44,33 +44,33 @@ benchmark [arguments (optional)]
...
+| micro-route              | 2.5.0   | ✓      | 41872.0    | 2.32    | 6.53          |
+| micro                    | 9.3.2   | ✗      | 45193.6    | 2.15    | 7.03          |
+| microrouter              | 3.1.3   | ✓      | 24536.0    | 3.98    | 3.85          |
+| polka                    | 0.5.1   | ✓      | 44393.6    | 2.19    | 6.97          |
+| rayo                     | 1.0.4   | ✓      | 44553.6    | 2.18    | 6.97          |
```
Comment (aichholzer) [inline README.md]: Current version is `1.2.5`

File: README.md [hunk 3]
```diff
@@ -44,33 +44,33 @@ benchmark [arguments (optional)]
...
+| restify                  | 7.2.1   | ✓      | 22968.0    | 4.08    | 3.64          |
```
Comment (aichholzer) [inline README.md]: Current version is `7.2.3`

**Comment by JamesKyburz** [comment]
Ok a manual npm install @latest it is! Was too lazy to do that
before...good catch :)

**Comment by JamesKyburz** [comment]
@aichholzer I just checked and the benchmarks where run with the latest.

However the table generated used the versions in package.json replacing the leading ^, i.e not the version installed

```javascript
const { dependencies } = require('../package')
 packages[pkg].version = version.replace('^', '')
```

I have now fixed this using require.resolve and hopefully the generated table will look better now :)

**Comment by aichholzer** [comment]
Thank you @JamesKyburz

**Pull request merged**